src/policy/fees.h

```diff
@@ -141,19 +207,38 @@ class CBlockPolicyEstimator
 
     /** Classes to track historical data on transaction confirmations */
     TxConfirmStats* feeStats;
+    TxConfirmStats* shortStats;
```

Comment:
This was the change made in #9942, and I couldn't get unique_ptrs to work while also moving the code to the cpp file.  I now also have other ideas how to clean up the wonky design of TxConfirmStats inside CBlockPolicyEstimator, but I don't want to do it now and churn this PR.